all-the-icons.el

```diff
@@ -1065,30 +1070,33 @@ When PFX is non-nil, ignore the prompt and just install"
   (interactive "P")
   (when (or pfx (yes-or-no-p "This will download and install fonts, are you sure you want to do this?"))
     (let* ((url-format "https://raw.githubusercontent.com/domtronn/all-the-icons.el/master/fonts/%s")
-           (font-dest (cond
-                       ;; Default Linux install directories
-                       ((member system-type '(gnu gnu/linux gnu/kfreebsd))
-                        (concat (or (getenv "XDG_DATA_HOME")
-                                    (concat (getenv "HOME") "/.local/share"))
-                                "/fonts/"))
-                       ;; Default MacOS install directory
-                       ((eq system-type 'darwin)
-                        (concat (getenv "HOME") "/Library/Fonts/"))))
-           (known-dest? (stringp font-dest))
-           (font-dest (or font-dest (read-directory-name "Font installation directory: " "~/"))))
+	   (font-dest (cond
+		       ;; Default Linux install directories
+		       ((member system-type '(gnu gnu/linux gnu/kfreebsd))
+			(concat (or (getenv "XDG_DATA_HOME")
```

Comment:
Please fix the whitespaces

Reply:
Thanks. Done!

Reply:
These are still tabs instead of spaces. Try `M-x whitespace-cleanup`.

Reply:
Just pushed some changes. I merged with the main before pushing so my changes are on the most up-to-date version.

Reply:
Clearly they are still tabs lol. You need to figure out what's happening with you editor

Reply:
Nooo. I received another email saying you merged something from my fork into the master. Do I still need to look at the tabs issue? I think I resolved it, and had pushed the code to the `master` branch in my fork. I thought this automatically pushes that code into my pull-request as well. 

Sorry for all the confusion, and thanks for your patience :smile: As you can see, this is my first time submitting anything to a public repo!